src/ua_parser/__init__.py

```diff
@@ -1 +1,83 @@
-VERSION = (0, 16, 1)
+"""The package provides top-level helpers which use a lazily initialised
+default parser. These are convenience functions, for more control it
+is perfectly acceptable to instantiate and call parsers directly.
+
+The default parser does use a cache keyed on the user-agent string,
+but its exact behaviour is unspecified, if you require a consistent
+behaviour or specific algorithm, set up your own parser (global or
+not).
+
+For convenience, direct aliases are also provided for:
+
+- :mod:`core types <.types>`
+- :mod:`caching utilities <.caching>`
+- :mod:`ua_parser.basic.Parser` as :class:`BasicParser`
+
+This way importing anything but the top-level package should not be
+necessary unless you want to *implement* a parser.
+"""
+VERSION = (1, 0, 0)
+
+from typing import Optional
+from .core import *
+from .basic import Parser as BasicParser
+from .caching import CachingParser, Clearing, LRU
+from .loaders import load_builtins, load_data, load_yaml
+
+
+parser: Parser
+
+
+def __getattr__(name):
+    global parser
+    if name == "parser":
+        parser = CachingParser(
+            BasicParser(load_builtins()),
+            LRU(200),
+        )
+        return parser
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+
+def parse(ua: str) -> ParseResult:
+    """Parses the :class:`.UserAgent`, :class:`.OS`, and :class:`.Device`
+    information using the :func:`global parser <get_parser>`.
+
+    Equivalent to calling each of :func:`parse_user_agent`,
+    :func:`parse_os`, and :func:`parse_device` but *may* be more
+    efficient than calling them separately depending on the underlying
+    parser.
+
+    Even in the best case, prefer the domain-specific helpers if
+    you're not going to use *all* of them.
+    """
+    from . import parser
```

Comment:
This is required to trigger `__getattr__` and actually initialise the parser, `global` completely fails to do that.